stock_reserve_rule/views/stock_reserve_rule_views.xml

```diff
@@ -62,6 +63,19 @@
                                         widget="many2many_tags"
                                         attrs="{'invisible': [('removal_strategy', '!=', 'packaging')]}"
                                     />
+                                    <field
+                                        name="tolerance_requested_limit"
+                                        attrs="{'invisible': [('removal_strategy', '!=', 'single_lot')], 'required': [('removal_strategy', '=', 'single_lot')]}"
```

Comment:
put required on the field, set default and drop required here

Reply:
Resolved